source/anaconda/index.rst

```diff
@@ -95,6 +95,12 @@ Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包、管理虚
 
    $ conda deactivate
 
+.. note::
```

Comment:
Why do you want to disable the base environment?

Reply:
Why do you want to activate it?

Reply:
Sometimes when I don't want to use python and open a shell, but there is `base` in the console.

Reply:
https://stackoverflow.com/questions/54429210/how-do-i-prevent-conda-from-activating-the-base-environment-by-default

A related question.

Reply:
done